### PR TITLE
[CDB-1167] CDB What's supported

### PR DESCRIPTION
**Pull request by brian-f-harness**
CDB What's supported

For reviewers, a preview is available.

[What's supported](https://64f0fcb62d2c0a3c1a6d21ed--harness-developer.netlify.app/docs/platform/Dashboards/cdb-whats-supported)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/18_Dashboards/cdb-whats-supported.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+This topic lists the supported dashboard features and integrations you can use in Harness.
+
+For a comprehensive list that includes all Harness modules, go to [Supported platforms and technologies](/docs/getting-started/supported-platforms-and-technologies).
+
+## Custom Dashboards Harness module support
+
+| Module | SaaS | Self-Managed Enterprise Edition |
+| :--: |  :--: | :--: |
+| CE | ✅ | ✅ |
```
Comment (Rohit-Kaliki) [inline docs/platform/18_Dashboards/cdb-whats-supported.md:15]: CE is supported only on SaaS

File: docs/platform/18_Dashboards/cdb-whats-supported.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+| CET| ✅ | ✅ |
```
Comment (Rohit-Kaliki) [inline docs/platform/18_Dashboards/cdb-whats-supported.md:16]: CET is supported only on SaaS

File: docs/platform/18_Dashboards/cdb-whats-supported.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| CI | ✅ | ✅ |
+| CCM| ✅ | ✅ |
```
Comment (Rohit-Kaliki) [inline docs/platform/18_Dashboards/cdb-whats-supported.md:18]: CCM SMP support is in Beta - can we call it out here?

File: docs/platform/18_Dashboards/cdb-whats-supported.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+| CD | ✅ | ✅ |
+| CV | ✅ | ✅ |
```
Comment (Rohit-Kaliki) [inline docs/platform/18_Dashboards/cdb-whats-supported.md:20]: CV is supported only on SaaS

File: docs/platform/18_Dashboards/cdb-whats-supported.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+| FF | ✅ | ✅ |
```
Comment (Rohit-Kaliki) [inline docs/platform/18_Dashboards/cdb-whats-supported.md:21]: FF SMP support is in Beta

File: docs/platform/18_Dashboards/cdb-whats-supported.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+| SEI|  |  |
+| SRM| ✅ | ✅ |
```
Comment (Rohit-Kaliki) [inline docs/platform/18_Dashboards/cdb-whats-supported.md:23]: SRM SMP support is in Beta

File: docs/platform/18_Dashboards/cdb-whats-supported.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+| STO| ✅ | ✅ |
```
Comment (Rohit-Kaliki) [inline docs/platform/18_Dashboards/cdb-whats-supported.md:24]: STO SMP support is in Beta

File: docs/platform/18_Dashboards/cdb-whats-supported.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+## Custom Dashboards Self-Managed Enterprise Edition support
+
+| **Connected/Air-gapped** | **Network ingress**         | **Certificate**      | **Supported** |
+| ----------------------- | ----------------------------- | -------------------- | ------------------------------- |
+| Connected               | Istio strict mode             | Self-signed          | ✅           |
+| Connected               | Istio strict mode             | Public          |            |
+| Connected               | Nginx             | Self-signed          | ✅           |
+| Connected               | Nginx             | Public          |            |
+| Air-gapped               | Istio strict mode             | Self-signed          | ✅           |
```
Comment (Rohit-Kaliki) [inline docs/platform/18_Dashboards/cdb-whats-supported.md:34]: For Airgapped, let's call out ✅ (Beta)

**Review by Rohit-Kaliki — CHANGES_REQUESTED** [review]

**Review by Rohit-Kaliki — APPROVED** [review]

File: docs/platform/18_Dashboards/cdb-whats-supported.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+| Air-gapped                | Nginx             | Self-signed          | ✅ (Beta)    |
+| Air-gapped               | Nginx             | Public          |            |
+
+## Custom dashboards support known limitations
+
+- CCM models have a query limit of 350GB per tile.
+
+- Self-Managed Enterprise Edition:
+
+   - Harness does not support FIPS compliant environments.
```
Comment (michaelcretzman) [inline docs/platform/18_Dashboards/cdb-whats-supported.md:45]:
```suggestion
   - Harness does not support FIPS-compliant environments.
```

**Review by michaelcretzman — DISMISSED** [review]
Approved with a minor suggestion.

**Pull request merged**